release/ray_release/glue.py

```diff
@@ -52,12 +51,11 @@
 )
 
 type_str_to_command_runner = {
-    "sdk_command": SDKRunner,
+    "job": JobRunner,
```

Comment:
I think I removed job by mistake at some point. Folks are still using job, not through ci/cd, but locally.